relationships/candidates/CONFLUENTCLOUDKAFKATOPIC.yml

```diff
@@ -0,0 +1,18 @@
+category: CONFLUENTCLOUDKAFKATOPIC
```

Comment:
Please, do not create a new category for an specific Kafka topic entity.
Add a new lookup to the general `KAFKATOPIC` category: https://github.com/newrelic/entity-definitions/blob/main/relationships/candidates/KAFKATOPIC.yml